src/logging/logging.cpp

```diff
@@ -112,9 +112,9 @@ void InitModules::init_modules(std::initializer_list< const char* > mods_list) {
 
 std::shared_ptr< spdlog::logger >& GetLogger() {
 #if __cplusplus > 201703L
-    [[unlikely]] if (!(logger_thread_ctx.m_logger)) {
+    if (!(logger_thread_ctx.m_logger)) {
 #else
-    if (LOGGING_PREDICT_FALSE(!(logger_thread_ctx.m_logger))) {
+    [[unlikely]] if (LOGGING_PREDICT_FALSE(!(logger_thread_ctx.m_logger))) {
```

Comment:
This seems backwards... from the code it appears only CPP > 2017.03 has `[[unlikely]]` support, but you've made it only for things < 2017.03 now. Do we just need a new > 2020.02 condition? I'm thinking this will break on <= c++14 now?

Reply:
To follow up; I think we've committed to only supporting >= C++20; so we can probably just remove support for <= 17

Reply:
agree, since we've committed to only supporting >= C++20, it makes sense to remove support for <= 17. let`s just remove this macro here